doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -1,1224 +1,1411 @@
+the time it was compiled.
 
-  The command tried to load library file {\ident}.vo that depends on
-  some specific version of library {\qualid} which is not the one
-  already loaded in the current {\Coq} session. Probably {\ident}.v
-  was not properly recompiled with the last version of the file
-  containing module {\qualid}.
 
-\item \errindex{Bad magic number}
+.. exn:: Require is not allowed inside a module or a module type
 
-  \index{Bad-magic-number@{\tt Bad Magic Number}}
-  The file {\tt{\ident}.vo} was found but either it is not a \Coq\
-  compiled module, or it was compiled with an older and incompatible
-  version of {\Coq}.
+This command
+is not allowed inside a module or a module type being defined. It is
+meant to describe a dependency between compilation units. Note however
+that the commands Import and Export alone can be used inside modules
+(see Section :ref:`TODO-2.5.8-import`).
 
-\item \errindex{The file {\ident}.vo contains library {\dirpath} and not
-  library {\dirpath'}}
 
-  The library file {\dirpath'} is indirectly required by the {\tt
-  Require} command but it is bound in the current loadpath to the file
-  {\ident}.vo which was bound to a different library name {\dirpath}
-  at the time it was compiled.
 
-\item \errindex{Require is not allowed inside a module or a module type}
+See also: Chapter :ref:`TODO-14-coq-commands`
 
-  This command is not allowed inside a module or a module type being defined.
-  It is meant to describe a dependency between compilation units. Note however
-  that the commands {\tt Import} and {\tt Export} alone can be used inside
-  modules (see Section~\ref{Import}).
 
-\end{ErrMsgs}
+.. cmd:: Print Libraries.
 
-\SeeAlso Chapter~\ref{Addoc-coqc}
+This command displays the list of library files loaded in the
+current |Coq| session. For each of these libraries, it also tells if it
+is imported.
 
-\subsection[\tt Print Libraries.]{\tt Print Libraries.\comindex{Print Libraries}}
 
-This command displays the list of library files loaded in the current
-{\Coq} session. For each of these libraries, it also tells if it is
-imported.
+.. cmd:: Declare ML Module {+ @string } .
 
-\subsection[\tt Declare ML Module {\str$_1$} .. {\str$_n$}.]{\tt Declare ML Module {\str$_1$} .. {\str$_n$}.\comindex{Declare ML Module}}
-This commands loads the {\ocaml} compiled files {\str$_1$} {\ldots}
-{\str$_n$} (dynamic link). It is mainly used to load tactics
-dynamically.
-% (see Chapter~\ref{WritingTactics}).
- The files are
-searched into the current {\ocaml} loadpath (see the command {\tt
-Add ML Path} in the Section~\ref{loadpath}).  Loading of {\ocaml}
-files is only possible under the bytecode version of {\tt coqtop}
-(i.e. {\tt coqtop.byte}, see chapter 
-\ref{Addoc-coqc}), or when {\Coq} has been compiled with a version of
-{\ocaml} that supports native {\tt Dynlink} ($\ge$ 3.11).
+This commands loads the Objective Caml compiled files 
```

Comment:
Why are these printed as "Objective Caml" while it was `\ocaml` in the LaTeX source? Oh no! The version that was used to translate this chapter is not even 8.7.2 but 8.7.0!